src/resources/formats/html/_quarto-rules.scss

```diff
@@ -106,6 +106,11 @@ figure .quarto-layout-row figure figcaption {
 .quarto-figure-right > figure > div /* for mermaid and dot diagrams */ {
   text-align: right;
 }
+
+.quarto-figure > figure > div code {
```

Comment:
Could we also add `.quarto-figure > figure > div.cell-annotation,` this to target this case:

````
```{r}
#| label: fig-slot
#| fig-cap: "The Big One"
#| fig-subcap: ["First", "Second"]
plot(cars)
plot(pressure) # <1> 
```
1. What is up phule
````

Reply:
I pushed a change that addresses this, thanks!